### PR TITLE
Add a markdown linter !

### PR DESCRIPTION
**Pull request by MoOx**
Ref #438

Now we need to choose rules we want & fix errors :)

See travis result to see what it looks like ;)

### CODE REVIEW

**Comment by MoOx** [comment]
w00t https://travis-ci.org/putaindecode/putaindecode.fr/builds/66495033

File: scripts/markdownlint.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const lintFail = new Error("markdown lint failed")
+
+async function lint() {
+  const results = await markdownlint({
+    files: [
+      ...await glob("*.md"),
+      ...await glob("src/**/*.md"),
+      ...await glob("content/**/*.md"),
+    ],
+  })
```
Comment (MoOx) [inline scripts/markdownlint.js]: THIS 😱❤️

**Comment by MoOx** [comment]
This might helps https://github.com/DavidAnson/markdownlint/blob/master/style/relaxed.json

**Comment by magsout** [comment]
Voila une rapide liste de ce que je considére à supprimer dans le linter

{
"MD002": false,
"MD022": false,
"MD024": false,
"MD012": false
}

**Comment by MoOx** [comment]
mdast-lint looks better.

**Pull request closed without merging**